tellme/views.py

```diff
@@ -22,7 +27,7 @@ def post_feedback(request):
                         'user': request.user.id}
         else:
             data = {'url': feedback['url'], 'browser': json.dumps(feedback['browser']), 'comment': feedback['note'],
-                        'email': feedback['email']}
+                        'email': feedback.get('email')}
```

Comment:
how is this change related ? And should it be done for the other fields of the form ?